docs-src/concepts/what-is-versioning.md

```diff
@@ -0,0 +1,170 @@
+
+## Worker Versioning
+
+Worker Versioning simplifies the deployment process of changes to Workflow Definitions by assigning a Build ID to the code that defines a Worker.
+
+In the context of deploying changes to Workflow Definitions in a distributed system, the Worker Versioning strategy has a number of advantages and disadvantages:
+
+### Advantages of Worker Versioning
+
+Worker Versioning is valuable in several respects:
+
+- **Safety against non-deterministic changes**: By providing a way to manage non-deterministic changes, Worker Versioning makes it less likely that Workers will attempt to process Workflow Tasks and Activity Tasks that they can't successfully handle.
+
+- **Greater flexibility**: You can make compatible changes to open Workflows by adding a new version to an existing Version Set and defining it as _compatible_ with an existing version.
+
+- **Smooth transition**: Allows for a smooth transition between different versions of Workers. New Workers execute new Workflow Executions, while old Workers with an appropriate version process old Workflow Executions.
+
+- **Controlled decommissioning of old Workers**: Worker Versioning strategy allows for controlled decommissioning of old Workers as they can be phased out after all Workflows using their version are closed.
+
+- **Integrated with Temporal's tooling**: Worker Versioning is managed through Temporal's CLI and SDKs, which makes it easier to integrate into your Workflows.
+
+### Disadvantages of Worker Versioning
+
+- **Long-lived Workflows**: While Worker Versioning works well with short-lived Workflows, it could be more challenging to implement with longer-lived Workflows. You might need to run multiple Worker versions simultaneously while open Workflows complete and are limited by the constraints of Worker Version Sets.
+
+- **Increased operational overhead**: There could be increased operational overhead in managing different versions of Workers and ensuring compatibility, particularly in larger systems with many Workers.
+
+### What is a Build ID?
+
+A Build ID is any string that you associate with a version of a Workflow. As a user-generated string, it's used to track, manage, and reference different stages or versions of the Workflow.
+
+In the context of a Workflow with Version Sets, a Build ID can be assigned as the default for a Version Set or as the overall default for the entire Workflow.
+
+A Build ID can contain a maximum of 255 characters.
+
+#### What is a default Build ID?
+
+A Default Build ID is the specific Build ID that a Worker refers to when executing Tasks.
+While this is typically the most recent Build ID, you can manually specify a different Build ID as the default when registering the Workflow with a Worker.
+This option is beneficial for situations where you prefer an older Build ID to be the default rather than the latest one.
+A unique Default Build ID can be assigned to both the Task Queue and each individual Compatible Version Set.
```

Comment:
This whole section isn't quite right. 

"you can manually specify a different Build ID as the default when registering the Workflow with a Worker." -- where did this idea come from? No such option exists. I just want to make sure we don't have misleading content somewhere else that made you think this.

There is a default for each compatible set, and there is the overall default, which is just the default of the default set - there is nothing else to consider.

When starting an activity/child/continue-as-new, you can decide if you want to stick to the compatible set of the invoking workflow, or the queue's default - that's the only place you have some kind of choice related to what version is used, and even then it's indirect.